### DISCUSSION TITLE
Running commands from text files / scripts

### DISCUSSION THREAD

**Discussion by Rnen** [Rnen/AdminToolbox#119] [Ideas]
### Describe the solution you'd like
Run commands from text files / scripts

**Comment by RainBawZ** [comment]
excellent

The way I imagine it:

Command is file name, so say 

> exampleCommand.txt

which contains commands like
```
role * 1
tpx * someplayer
```

Another neat thing would be if scripts could recieve arguments. Something like
```
#args=$arg1;$arg2$;arg3;
#help="Commandname [arg1] [arg2] [arg3]";
```
Where the string specified in #help would display if the command is called with invalid arguments
The command would be called like this:
`exampleCommand arg1 arg2 arg3`

So a full example script would look like:
```
#args=$player1;$player2;$role;
#help="ExampleCommand [Player1] [Player2] [Role]";
role $player1 $role
tpx $player2 $player1
```
